doc/modules/model_evaluation.rst

```diff
@@ -473,6 +473,13 @@ given binary ``y_true`` and ``y_pred``:
       for C-class classification problem).
     * Macro-average recall as described in [Mosley2013]_ and [Kelleher2015]_: the recall
       for each class is computed independently and the average is taken over all classes.
+    * Class balanced accuracy as described in [Mosley2013]_: for each class, the number
+      of correctly predicted samples (diagonal element in the confusion matrix) is normalized
+      by the maximum value of either the total number of observations predicted to the class
+      (sum of the class' column in the confusion matrix) or the actual number
+      of observations in that class (sum of the class' row in the confusion matrix).
+      In other words, we take the minimum between the precision and the recall for each class.
```

Comment:
Do you think stating the definition from confusion matrix marginals here but not in the macro recall case is helpful?

Reply:
You're right, either I should put it in both definitions or in none. Does not make sense to put it just in one of them. I'll delete it then 👍

Reply:
Do you know what the benefit of accounting for false positives (and hence precision) is? This seems to make it a different metric that no longer reduces to balanced accuracy in the binary case..?

Reply:
I also generally think that having a denominator which depends on the predictions is against the whole philosophy of ROC and BalAcc-style metrics.